docs/core/tools/global-tools-how-to-create.md

```diff
@@ -0,0 +1,197 @@
+description: Describes how to create a Global Tool. The Global Tool is a console application that is installed through the .NET Core CLI.
+author: Thraka
+ms.author: adegeo
+ms.date: 08/22/2018
+---
+
+# Create a .NET Core Global Tool
+
+This article teaches you how to create and package a .NET Core Global Tool. The .NET Core CLI allows you to create a console application as a Global Tool, which others can easily install. .NET Core Global Tools are NuGet packages that are installed from the .NET Core CLI. For more information about Global Tools, see [.NET Core Global Tools overview][global-tool-info].
+
+[!INCLUDE [topic-appliesto-net-core-21plus.md](../../../includes/topic-appliesto-net-core-21plus.md)]
+
+## Create a project
+
+This article uses the .NET Core CLI to create and manage a project. Obviously you could use Visual Studio to do some of these steps, but Visual Studio will not be described in this article.
+
+Our example tool will be a console application that generates an ascii bot and prints a message. First, create a new .NET Core 2.1 Console Application.
+
+```console
+dotnet new console -o botsay
+```
+
+Enter the `botsay` directory created by the previous command.
+
+## Add the code
+
+Open the `Program.cs` file with your favorite text editor, such as `vim` or [Visual Studio Code](https://code.visualstudio.com/).
+
+Add the following `using` directive to the top of the file, this will help to display the version information of the application.
+
+```csharp
+using System.Reflection;
+```
+
+Next, move down to the `static void Main(string[] args)` method. Erase any code that exists inside this method, most likely you have a print statement in there from the console application template.
+
+Add the following code to process the command line arguments for your application. This code will first check if any arguments were passed. If no arguments were passed, a little help message will be displayed. If arguments were passed, all of those arguments will be transformed into a string and printed with the bot.
+
+```csharp
+static void Main(string[] args)
+{
+    if (args.Length == 0)
+    {
+        var versionString = Assembly.GetEntryAssembly()
+                                .GetCustomAttribute<AssemblyInformationalVersionAttribute>()
+                                .InformationalVersion
+                                .ToString();
+                                
+        Console.WriteLine($"botsay v{versionString}");
+        Console.WriteLine("-------------");
+        Console.WriteLine("\nUsage:");
+        Console.WriteLine("  botsay <message>");
+        return;
+    }
+
+    ShowBot(string.Join(' ', args));
+}
+```
+
+### Create the bot
+
+Next, add a new method named `ShowBot` which takes a string parameter. This method will print out the message and the ascii bot. The ascii bot code was taken from the awesome [dotnetbot](https://github.com/dotnet/core/blob/master/samples/dotnetsay/Program.cs) sample.
```

Comment:
remove future tense

check capitalization of ascii

which -> that?

awesome dotnet sample sounds like marketing speak. remove awesome IMO